charts/dex.yaml

```diff
@@ -0,0 +1,56 @@
+image: '{%- endraw -%}{{ build_image_name(\"dex\", False) }}{%- raw -%}'
+
+nodeSelector:
+  node-role.kubernetes.io/infra: ''
+
+tolerations:
+  - key: "node-role.kubernetes.io/bootstrap"
+    operator: "Exists"
+    effect: "NoSchedule"
+  - key: "node-role.kubernetes.io/infra"
+    operator: "Exists"
+    effect: "NoSchedule"
+
+replicas: 2
+
+# grpc support
+grpc: false
+
+# https termination by dex itself
+https: false
+
+ingress:
+  enabled: true
+  annotations:
+    nginx.ingress.kubernetes.io/rewrite-target: '/$2'
+    nginx.ingress.kubernetes.io/use-regex: "true"
+    nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
+    kubernetes.io/ingress.class: "nginx-control-plane"
+  path: /oidc(/|$)(.*)
+  hosts:
+    - null
+  tls:
+   - secretName: dex-tls-secret
+
+certs:
+  web:
+    create: false
+  grpc:
+    create: false
+
+config:
+  issuer: https://{{ pillar['metalk8s']['api_server']['host'] }}:32000
+
+  web:
+    # port is taken from ports section above
+    address: 0.0.0.0
```

Comment:
Shouldn't it listen on the control-plane IP only, as suggested in #2011?

Reply:
Huh, no.

This stuff is running in a container, not in the host network, so binding to a control-plane IP (which IP would that be, by the way? Of which node?) can't work.

`0.0.0.0` is fine and will basically bind to `lo` and the only other interface in the container.

However, again, isn't this the default in the chart's `values`, so do we really need to set it here?!

Reply:
it's default in the charts.. taking it off